core/test/Node.js

```diff
@@ -236,6 +236,19 @@ test('Node', function(t) {
         t.end();
     });
 
+    t.test('getPositionFrom method', function(t) {
+        var root  = (new Node())    .setPosition(0,  0,  0);
+        var node1 = root.addChild() .setPosition(10, 10, 10);
+        var node2 = node1.addChild().setPosition(20, 20, 20);
+        var node3 = node2.addChild().setPosition(5,  5,  5);
+
+        var pos = node3.getPositionFrom(node1);
+
+        t.equal(pos[0], 25);
+        t.equal(pos[1], 25);
+        t.equal(pos[2], 25);
```

Comment:
should be in `/core/test/Node.spec.js` IMO.

Did you test this locally? This test shouldn't work.